### ISSUE TITLE
Course Materials - updating release date of file after setting release date of folder is confusing

### ISSUE THREAD

**Issue by bmcutler** [Submitty/Submitty#9736]
**Describe the bug**
Instructor uploaded a file inside of a folder in course materials.
Then set the release date of the whole folder.
Then went to re-edit the release date of the folder.

The UI was confusing.  It didn't save/update the release date of the file as expected.

I think this is because the "apply recursive updates" button was greyed out.

Also it's confusing that the dates on the main course materials page (the green box) look like you can type in them (you can place the cursor there) but you cannot edit the date here.
<img width="487" alt="Screenshot 2023-08-29 at 1 34 07 PM" src="https://github.com/Submitty/Submitty/assets/5871417/8c8be1f4-9c5a-456d-ac12-548e53c08fe1">

Bottom line, let's take another look at the GUI and the interaction and see if there are indeed bugs, or what can be clarified/simplified.
